pyproject.toml

```diff
@@ -30,4 +30,9 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.isort]
-profile = "black"
\ No newline at end of file
+profile = "black"
+
+[project]
+dependencies = [
+    "mkdocstrings[python]>=0.18",
```

Comment:
Does this actually create a new dependency? As far as I recall, I've so far avoided actually including development dependencies in the pyproject.toml. Even if they aren't installed, they still have to be solved for.